pico/pom.xml

```diff
@@ -0,0 +1,91 @@
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.helidon</groupId>
+        <artifactId>helidon-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>io.helidon.pico</groupId>
+    <artifactId>helidon-pico-project</artifactId>
+    <name>Helidon Pico Project</name>
+
+    <packaging>pom</packaging>
+
+    <properties>
+        <!-- all pico modules should be backward compatible with 11 -->
+        <version.java>11</version.java>
+
+        <!-- pico supports legacy javax.inject and javax.annotations in tooling -->
+        <javax.injection.version>1</javax.injection.version>
+<!--        <javax.annotations.version>1.3.2</javax.annotations.version>-->
+    </properties>
+
+    <!-- this will show the progression of what has come over, and what is left to come over -->
+    <modules>
+        <module>api</module>
+<!--        <module>spi</module>-->
+        <module>types</module>
+<!--        <module>builder</module>-->
+<!--        <module>config</module>-->
+<!--        <module>pico</module>-->
+<!--        <module>tck</module>-->
+<!--        <module>tools</module>-->
+<!--        <module>processor</module>-->
+<!--        <module>maven-plugin</module>-->
+<!--        <module>test-support</module>-->
+<!--        <module>test-utils</module>-->
+<!--        <module>test-resources</module>-->
+<!--        <module>test-extended-resources</module>-->
+<!--        <module>test-pico</module>-->
+<!--        <module>test-tck-jsr330</module>-->
+<!--        <module>examples</module>-->
+    </modules>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <configuration>
+                        <!-- This module is Java 11, we must remove enable-preview from parent project -->
+                        <additionalOptions combine.self="override"/>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+    <developers>
```

Comment:
This only belongs to root pom file of Helidon, we do not have any other developer attributions (git is the source of truth)